backend/capellacollab/settings/modelsources/t4c/repositories/interface.py

```diff
@@ -61,3 +64,56 @@ def stop_repository(instance: DatabaseT4CInstance, name: str):
     )
     r.raise_for_status()
     return r.json()
+
+
+def get_t4c_status(instance: DatabaseT4CInstance):
```

Comment:
Is there a reason for having the same code twice? In `capellacollab/projects/capellamodels/modelsources/t4c/connection.py`, there is exactly the same code.

Reply:
In addition, `fetch_last_seen` should be moved as well.